covalent/executor/executor_plugins/dask.py

```diff
@@ -85,6 +94,9 @@ def __init__(
         super().__init__(log_stdout, log_stderr, conda_env, cache_dir, current_env_on_conda_fail)
 
         self.scheduler_address = scheduler_address
+        self.workdir = workdir
+        set_config("executors.dask.workdir", self.workdir)
+        Path(self.workdir).mkdir(parents=True, exist_ok=True)
```

Comment:
Can you move this into the `run` method instead? Ideally we would like postpone the creation of the directory as close to its need as possible.

Reply:
done